core/CMakeLists.txt

```diff
@@ -88,7 +88,11 @@ include(SetupCCTZ)
 find_package_required(Http_Parser "libhttp-parser-dev")
 
 add_library(${PROJECT_NAME} STATIC ${SOURCES})
-target_compile_definitions(${PROJECT_NAME} PRIVATE SPDLOG_PREVENT_CHILD_FD)
+if (fmt_FOUND)
```

Comment:
That's starnge. I thought that `fmt_FOUND` is always defined.

Can we just always define `SPDLOG_FMT_EXTERNAL` for all the platforms?